common/src/main/resources/assets/cloth-config2/lang/en_us.json

```diff
@@ -33,6 +33,7 @@
     "text.cloth-config.boolean.value.true": "§aYes",
     "text.cloth-config.boolean.value.false": "§cNo",
     "text.cloth-config.dropdown.value.unknown": "§cNo suggestions",
+    "text.cloth-config.disabled_tooltip": "Disabled (requirements not met)",
```

Comment:
Should this have any formatting? Colour? :thinking: 

![image](https://github.com/shedaniel/cloth-config/assets/5046562/df63200d-a999-43ab-be03-6e373df9edfa)

Reply:
I think this is fine